### ISSUE TITLE
Bulk Edit doesn't include description

### ISSUE THREAD

**Issue by zgirod** [a85/POSTMan-Chrome-Extension#9303]
**Is your feature request related to a problem? Please describe.**
When bulk editing the params for a request the bulk edit includes the key/value however the description is not available in the bulk edit.

**Describe the solution you'd like**
I would like description to be included in bulk editing params for a request.

**Comment by Bentzeh** [comment]
looking forward to this update as well

**Comment by Eieyron** [comment]
bump

**Comment by ryanilg** [comment]
Bump!

**Comment by darmenante** [comment]
Bump!

**Comment by arunrreddy** [comment]
Is there any update on this feature? It has been in a pending triage state for way too long.

**Comment by Silky777** [comment]
It would be extremely helpful to have descriptions available through Bulk Edit. It really doesn't make any sense that this hasn't been added after this long.

**Comment by HAYVENO** [comment]
It's 2025, and I am here to remind the team about the famously-requested **Description bulk-edit feature.**

**Comment by ThioJoe** [comment]
Yes please, especially in the age of AI where we can have it generate a whole template, it would be nice to let it be able to add descriptions too
